### DISCUSSION TITLE
Potentional license violation?

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#1855] [General]
Found a fork that pretends to be not-a-fork of fooocus, imported from scratch it seems, commit log is wiped. So far noticed no major differences.
https://github.com/BlueMoonAI/BlueMoonAI

**Comment by mashb1t** [comment]
wtf

**Reply by mashb1t** [reply]
looking into this, @aolko thank you for the hint.
@lllyasviel fyi

**Reply by mashb1t** [reply]
aaand it's gone.
There are additional mirrors though:
- https://github.com/KamiyaKatsumi/BlueMoonAI
- https://github.com/MorikawaAsami/BlueMoonAI

**Reply by aolko** [reply]
also gone

**Reply by aolko** [reply]
fyi [the internet remembered](https://web.archive.org/web/*/https://github.com/BlueMoonAI/BlueMoonAI)

**Comment by lllyasviel** [comment]
gpl allows changing anything including any name, any title, any logo, with or without any attributions, with or without any author names, as long as the codes and gpl lisence file is there.

And this is also how fooocus works when we modify codes from auto, comfyui, diffusers, k-diffusion, etc.

**Reply by mashb1t** [reply]
yeah, that's why "wtf" was the first response, as they use(d?) an "Apache2" license. You can still see this in https://github.com/MorikawaAsami/BlueMoonAI (also has an OpenRail license somehow, see [here](https://github.com/MorikawaAsami/BlueMoonAI/blob/main/LICENSE.md)), but https://github.com/KamiyaKatsumi/BlueMoonAI has changed this to "GNU Affero General Public License v3.0" but also said OpenRail license.

**Reply by aolko** [reply]
> yeah, that's why "wtf" was the first response, as they use(d?) an "Apache2" license. You can still see this in https://github.com/MorikawaAsami/BlueMoonAI (also has an OpenRail license somehow, see [here](https://github.com/MorikawaAsami/BlueMoonAI/blob/main/LICENSE.md)), but https://github.com/KamiyaKatsumi/BlueMoonAI has changed this to "GNU Affero General Public License v3.0" but also said OpenRail license.

Don't forget copyrights

**Reply by mashb1t** [reply]
plus that, very  many files include(d) copyright and claim ownership of the code exclusively by BlueMoonAI

**Comment by thiner** [comment]
Interesting, the repo, including folk repos are disappeared completely in just 18 hours. Is that done by Github?

**Reply by mashb1t** [reply]
Either this or they have renamed their repositories/accounts. Their Patreon and Website are both also not available anymore, which is why i assume the latter.

**Comment by aolko** [comment]
Found this in my mail
![](https://i.ibb.co/yf5Qbf2/chrome-Mxs9-K0-Ul-Ox.png)

Lead to https://medium.com/@samartrew/bluemoon-ai-a-new-open-source-stable-diffusion-model-a2989ffa5a4a
which redirects to `https://medium.com/@fzndrdstdofficial/bluemoon-ai-a-new-open-source-stable-diffusion-model-a2989ffa5a4a`

[[But the internet never forgets]](https://web.archive.org/web/20240106145604/https://medium.com/@fzndrdstdofficial/bluemoon-ai-a-new-open-source-stable-diffusion-model-a2989ffa5a4a)

[[The account is a little weird too, probably a sockpuppet]](https://web.archive.org/web/20240106145630mp_/https://medium.com/@samartrew)

***The plot thickens...***

**Reply by mashb1t** [reply]
i don't know what they had in mind but if Fooocus ever needs an advertisement text - this is it!

**Comment by aolko** [comment]
Managed to fish out the source
[BlueMoonAI-main.zip](https://github.com/lllyasviel/Fooocus/files/13907049/BlueMoonAI-main.zip)

**Reply by mashb1t** [reply]
i'd suggest not to redistribute their project code

**Reply by aolko** [reply]
\* for auditing purposes
